sdk/core/core/samples/README.md

```diff
@@ -7,7 +7,7 @@ products:
 urlFragment: core-samples
 ---
 
-# Azure Core Samples client library for C
+# Azure Core Samples client Library for Embedded C
```

Comment:
Why capitalize  the L in library, and if we are doing it also capitalize the C in client?

Reply:
The name doesn't make a ton of sense.  Samples are never packaged as libraries.  I'll clean up these readme's in one of my subsequent readme cleanup PRs